elytron/src/test/resources/org/wildfly/extension/elytron/tls-test.xml

```diff
@@ -23,21 +23,25 @@
     </credential-stores>
     <tls>
         <key-stores>
-            <key-store name="FireflyKeystore" type="JKS">
-                <file path="firefly.keystore" relative-to="jboss.server.config.dir"/>
+            <key-store name="FireflyKeystore" >
                 <credential-reference store="test" alias="master-password-alias"/>
+                <implementation type="JKS" />
```

Comment:
type is mandatory when implementation element is present.

also it could make sense to have type default to "JKS" as I think in most scenarios this is what users will configure anyway.